README.md

```diff
@@ -189,7 +189,8 @@ Key bindings can be customized, see [configuration](readme/customization.md#key-
 | `e`         | Normal/Diff | Set selected commit(s) to be edited       |
 | `s`         | Normal/Diff | Set selected commit(s) to be squashed     |
 | `f`         | Normal/Diff | Set selected commit(s) to be fixed-up     |
-| `d`         | Normal/Diff | Set selected commit(s) to be dropped      |
+| `d`         | Normal      | Set selected commit(s) to be dropped      |
+| `d`         | Diff        | Show full commit diff                     |
```

Comment:
Shouldn't the complete list of keybindings be sorted for clarity?

Reply:
There used to be a natural order to these keybindings, where similar actions were grouped together. However, overtime, as functionality evolved, this grouping started to have less meaning.

I think this whole table needs a refresh, with the different modes explained.